src/mixins/base-transition.js

```diff
@@ -18,6 +18,10 @@ export const baseTransition = {
 			validator: validateDelay,
 			default: defaults.transitionDelay,
 		},
+		opacity: {
```

Comment:
Lets set the prop name to `no-opacity` and reverse the default value to fit widely used pattern `default-false`
Related: https://eslint.vuejs.org/rules/no-boolean-default.html